aip/general/0157.md

```diff
@@ -76,23 +75,13 @@ enum BookView {
   conflict. In this situation, the service **should** nest the view enum within
   the individual resource.
 
-### Read masks as a request field
-
-Similar to a system parameter, an API **may** support read masks as a single field on the request
-message: `google.protobuf.FieldMask read_mask`.
+### Read masks as a request field (deprecated)
```

Comment:
Now I see why you put `(recommended)` above. Hmm... is there a common pattern for deprecating parts of AIPs. Maybe this is the way?

Reply:
The only reference I saw to deprecation was on http://go/aip/client-libraries/4231#backwards-compatibility (last sentence about ORIGINALLY_SINGLE_PATTERN) and IMO, is not very obvious.

I was looking for other markdown that could be used, but didn't find anything obvious.